src/domains/contact/pages/Contacts/Contacts.test.tsx

```diff
@@ -346,15 +346,15 @@ describe("Contacts", () => {
 			);
 		});
 
-		it("should render empty", () => {
+		it("should render", () => {
```

Comment:
`should render` as name twice? 🤔

Reply:
Yea, since they are still unique in the context of their respective describe blocks (with contacts / without contacts). We can keep the more explicit naming if you prefer.

Reply:
yeah please use distinct names, makes it easier to figure out which test does what and if there is an issue with one of them

Reply:
i did some housekeeping on that file :oleglala: